app/init.php

```diff
@@ -1511,6 +1513,19 @@ function (mixed $value) {
     return new Cache(new Sharding($adapters));
 }, ['pools']);
 
+App::setResource('letsEncryptCertificateAdapter', function () {
```

Comment:
Very complex naming conventions: letsEncryptCertificateAdapter, adapterForCertificates. If we don't use letsEncryptCertificateAdapter resource anywhere directly, no point in having it as a separate resource, unnecessary noise. `adapterForCertificates` can just be `certificates`